src/app/pages/status/status.component.ts

```diff
@@ -103,4 +105,28 @@ export class StatusPage implements OnInit {
       }
     })
   }
+
+  interpretStatus(status: string): string {
+    switch (status) {
+      case 'completed':
```

Comment:
best practice for constant strings is to use all uppercase to make it more agnostic to potential readability issues in your text editor.

COMPLETED instead of completed

Reply:
Fixed.